src/util/updateNetworks.js

```diff
@@ -0,0 +1,27 @@
+import { rpc, settings } from '@cityofzion/neon-js';
+import { keys } from 'lodash';
+
+import { NOS_LOCAL, PREDEFINED_NETWORKS } from '../values/networks';
+
+const { networks } = settings;
+
+const nosLocalConfig = {
+  name: NOS_LOCAL,
+  extra: {
+    neoscan: 'http://localhost:4000/api/main_net'
+  }
+};
+
+export default function updateNetworks(userNetworks) {
+  keys(networks).forEach((name) => {
```

Comment:
Do you mean this to be `keys(userNetworks)`?

Reply:
No, my intention here was to remove all previous user networks first, then re-add them.  If there's an easier way (e.g.: maybe neon-js will justreplace an existing network name if we call `addNetwork`?), then I'm in favor of doing that instead.